code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -13,6 +13,42 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 	. = ..()
 	password = "[GLOB.fscpassword]"
 
+/obj/machinery/vending/medical/syndicate_access/cybersun
+	name = "\improper CyberMed ++"
+	desc = "Advanced vendor dispensing medical drugs."
```

Comment:
Description seems bland, maybe this could be okay?
```suggestion
	desc = "An advanced vendor that dispenses medical drugs, both recreational and medicinal."
```